src/Functions/multiIf.cpp

```diff
@@ -189,9 +193,19 @@ class FunctionMultiIf final : public FunctionIfBase</*null_is_false=*/true>
                 break;
         }
 
-        size_t rows = input_rows_count;
         MutableColumnPtr res = return_type->createColumn();
 
+        /// Special case if first instruction condition is always true and source is constant
```

Comment:
What if it's always false?

Reply:
If its always false it will be skipped 

```
Field value = column_const->getField();

if (value.isNull())
    continue;
if (value.get<UInt64>() == 0)
    continue;
```